UberSignals.xcodeproj/project.pbxproj

```diff
@@ -376,7 +487,7 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = "com.uber.$(PRODUCT_NAME:rfc1034identifier)";
-				PRODUCT_NAME = "$(TARGET_NAME)";
+				PRODUCT_NAME = UberSignals;
```

Comment:
I had to change the target names with `UberSignals iOS` & `UberSignals OSX` . But I decided not to update the product's name and stay with `UberSignals` for iOS and OS X.